libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/installer/ComponentInstaller.kt

```diff
@@ -218,3 +260,24 @@ class RouteArrowComponentConfig internal constructor(context: Context) {
      */
     var options = RouteArrowOptions.Builder(context).build()
 }
+
+/**
+ * Location puck component configuration class.
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class LocationPuckComponentConfig internal constructor() {
```

Comment:
What's the reason for using `Config` again?

Most places we use the `Options` pattern and `libnavui-androidauto` becomes inconsistent if it would use this

Reply:
Resolving as the installer makes use of this in a way that is still different than the Options